README.md

```diff
@@ -81,26 +81,100 @@ If you have any questions about ProtonPlus or want to share information with us,
 
 - - - -
 
-### 🛠️ Building from Source
+## 🛠️ Building from Source
 
 _Requirements_
 - [git](https://github.com/git/git)
 - [ninja](https://github.com/ninja-build/ninja)
-- [meson](https://github.com/mesonbuild/meson)
+- [meson >= 0.59.0](https://github.com/mesonbuild/meson)
 - [gtk4](https://gitlab.gnome.org/GNOME/gtk/)
-- [libadwaita-1](https://gitlab.gnome.org/GNOME/libadwaita)
-- [json-glib-1.0](https://gitlab.gnome.org/GNOME/json-glib)
-- [libsoup-3.0](https://gitlab.gnome.org/GNOME/libsoup)
+- [libadwaita >= 1.2](https://gitlab.gnome.org/GNOME/libadwaita)
+- [json-glib](https://gitlab.gnome.org/GNOME/json-glib)
+- [libsoup](https://gitlab.gnome.org/GNOME/libsoup)
 - [libarchive](https://github.com/libarchive/libarchive)
 - [desktop-file-utils](https://gitlab.freedesktop.org/xdg/desktop-file-utils)
 
-**Installing**
-
-- Install all dependencies (I'm using Fedora 37): `sudo dnf install git ninja-build meson gtk4-devel libadwaita-devel json-glib-devel libsoup3-devel libarchive-devel desktop-file-utils`
-- Clone the GitHub repo: `git clone https://github.com/Vysp3r/ProtonPlus.git` & `cd ProtonPlus`
-- Build the source: `meson build --prefix=/usr` & `cd build` & `ninja`
-- (Optional) Install application: `ninja install`
-- Start the application: `cd src` & `./protonplus`
+### Fedora
+
+1. Install all dependencies:
+    ```bash
+    sudo dnf install \
+      git \
+      ninja-build \
+      meson \
+      gtk4-devel \
+      libadwaita-devel \
+      json-glib-devel \
+      libsoup3-devel \
+      libarchive-devel \
+      desktop-file-utils
+    ```
+
+2. Clone the GitHub repo and change to repo directory:
+    ```bash
+    git clone https://github.com/Vysp3r/ProtonPlus.git && \
+      cd ProtonPlus
+    ```
+
+3. Build the source:
+    ```bash
+    meson build --prefix=/usr && \
+    cd build && \
+    ninja
+    ```
+
+4. (Optional) Install application:
+    ```bash
+    ninja install
+    ```
+
+5. Start application:
+    ```bash
+    cd src && \
+    ./protonplus
```

Comment:
Rename 'protonplus' to 'com.vysp3r.ProtonPlus' since I changed the exec name to that.